vulntotal/datasources/vulnerablecode.py

```diff
@@ -85,3 +85,26 @@ def parse_advisory(fetched_advisory) -> VendorData:
     return VendorData(
         aliases=aliases, affected_versions=affected_versions, fixed_versions=fixed_versions
     )
+
+
+class VCIOTokenError(Exception):
+    pass
+
+
+def fetch_vulnerablecode_query(url: str, payload: dict):
+    load_dotenv()
+    vcio_token = os.environ.get("VCIO_TOKEN", None)
+    if not vcio_token:
+        msg = "Cannot call VulnerableCode API without a token set in the VCIO_TOKEN environment variable."
+        raise VCIOTokenError(msg)
+
+    response = (
+        requests.post(url, headers={"Authorization": vcio_token}, json=payload)
```

Comment:
Add a comment showing what is expected in token in .env , AFAIK either the user can store just the token and you have to do something like this in that case `headers={"Authorization": "Token f{vcio_token}"}` or add a comment explaining the env should have something like this : `Token <YOUR_VCIO_TOKEN>`